tests/unit/dataactcore/test_populate_subaward_table.py

```diff
@@ -0,0 +1,635 @@
+        'sub_high_comp_officer1_amount': sub_grant.top_paid_amount_1,
+        'sub_high_comp_officer2_full_na': sub_grant.top_paid_fullname_2,
+        'sub_high_comp_officer2_amount': sub_grant.top_paid_amount_2,
+        'sub_high_comp_officer3_full_na': sub_grant.top_paid_fullname_3,
+        'sub_high_comp_officer3_amount': sub_grant.top_paid_amount_3,
+        'sub_high_comp_officer4_full_na': sub_grant.top_paid_fullname_4,
+        'sub_high_comp_officer4_amount': sub_grant.top_paid_amount_4,
+        'sub_high_comp_officer5_full_na': sub_grant.top_paid_fullname_5,
+        'sub_high_comp_officer5_amount': sub_grant.top_paid_amount_5,
+
+        'prime_id': grant.id,
+        'internal_id': grant.internal_id,
+        'date_submitted': grant.date_submitted.strftime('%Y-%m-%d %H:%M:%S.%f'),
+        'report_type': None,
+        'transaction_type': None,
+        'program_title': None,
+        'contract_agency_code': None,
+        'contract_idv_agency_code': None,
+        'grant_funding_agency_id': grant.funding_agency_id,
+        'grant_funding_agency_name': grant.funding_agency_name,
+        'federal_agency_name': grant.federal_agency_name,
+        'treasury_symbol': None,
+        'dunsplus4': grant.dunsplus4,
+        'recovery_model_q1': None,
+        'recovery_model_q2': None,
+        'compensation_q1': str(grant.compensation_q1).lower(),
+        'compensation_q2': str(grant.compensation_q2).lower(),
+        'high_comp_officer1_full_na': grant.top_paid_fullname_1,
+        'high_comp_officer1_amount': grant.top_paid_amount_1,
+        'high_comp_officer2_full_na': grant.top_paid_fullname_2,
+        'high_comp_officer2_amount': grant.top_paid_amount_2,
+        'high_comp_officer3_full_na': grant.top_paid_fullname_3,
+        'high_comp_officer3_amount': grant.top_paid_amount_3,
+        'high_comp_officer4_full_na': grant.top_paid_fullname_4,
+        'high_comp_officer4_amount': grant.top_paid_amount_4,
+        'high_comp_officer5_full_na': grant.top_paid_fullname_5,
+        'high_comp_officer5_amount': grant.top_paid_amount_5,
+        'sub_id': sub_grant.id,
+        'sub_parent_id': sub_grant.parent_id,
+        'sub_federal_agency_id': sub_grant.federal_agency_id,
+        'sub_federal_agency_name': sub_grant.federal_agency_name,
+        'sub_funding_agency_id': sub_grant.funding_agency_id,
+        'sub_funding_agency_name': sub_grant.funding_agency_name,
+        'sub_funding_office_id': None,
+        'sub_funding_office_name': None,
+        'sub_naics': None,
+        'sub_cfda_numbers': sub_grant.cfda_numbers,
+        'sub_dunsplus4': sub_grant.dunsplus4,
+        'sub_recovery_subcontract_amt': None,
+        'sub_recovery_model_q1': None,
+        'sub_recovery_model_q2': None,
+        'sub_compensation_q1': str(sub_grant.compensation_q1).lower(),
+        'sub_compensation_q2': str(sub_grant.compensation_q2).lower(),
+    }
+    if debug and not (attr.items() <= sub.__dict__.items()):
```

Comment:
Same debug question as above.

Reply:
After discussion, we decided to keep these in just for debugging purposes and it won't populate the logs.